Mesh_3/examples/Mesh_3/mesh_3D_weighted_image.cpp

```diff
@@ -1,3 +1,5 @@
+//#define CGAL_MESH_3_WEIGHTED_IMAGES_DEBUG
+
```

Comment:
You should probably remove those those lines, so that they do not appear in the user manual.
```suggestion
```